src/NServiceBus.Core/OpenTelemetry/Metrics/Metrics.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NServiceBus;
+
+static class Metrics
```

Comment:
Should all the constants names in this class include `Message` in their names?

Reply:
It makes sense to me to include message, even if "messaging" is already part of the name.